src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigClientImplTest.java

```diff
@@ -107,6 +108,20 @@ public void testGetTemplate() throws Exception {
     assertTrue(
             headerParameter.getDefaultValue() instanceof ParameterValue.InAppDefault);
     checkGetRequestHeader(interceptor.getLastRequest());
+
+    // Check Conditions
+    List<Condition> actualConditions = template.getConditions();
+    List<Condition> expectedConditions = ImmutableList.of(
+            new Condition("ios_en", "device.os == 'ios' && device.country in ['us', 'uk']")
+                    .setTagColor(TagColor.INDIGO),
+            new Condition("android_en",
+                    "device.os == 'android' && device.country in ['us', 'uk']")
+                    .setTagColor(TagColor.UNSPECIFIED)
+    );
+    assertEquals(expectedConditions.size(), actualConditions.size());
+    for (int i = 0; i < expectedConditions.size(); i++) {
+      assertEquals(expectedConditions.get(i), actualConditions.get(i));
+    }
```

Comment:
Add some explicit test cases for equality test between different conditions.

Reply:
Good call! Added a new test class `ConditionTest.java`